judge/views/submission.py

```diff
@@ -531,6 +531,21 @@ def get(self, request, *args, **kwargs):
         return super(ForceContestMixin, self).get(request, *args, **kwargs)
 
 
+class UserAllContestSubmissions(ForceContestMixin, AllUserSubmissions):
+    def get_title(self):
+        return "%s's submissions in %s" % (self.username, self.contest.name)
+
+    def access_check(self, request):
+        super(UserAllContestSubmissions, self).access_check(request)
+        if not self.contest.users.filter(user_id=self.profile.id).exists():
+            raise Http404()
+
+    def get_content_title(self):
+        return format_html(_('<a href="{1}">{0}</a>\'s submissions in <a href="{3}">{2}</a>'),
```

Comment:
I think @quantum5 has some gripes about HTML in translation strings, not sure if that applies here.

Reply:
There are examples such as https://github.com/DMOJ/online-judge/blob/master/judge/views/ticket.py#L87-L90, but maybe we should do this in a separate PR and get it all out of the way.